examples/example-vite-react-sdk/src/typescript/contracts.gen.ts

```
@@ -18,7 +18,7 @@ export function setupWorld(provider: DojoProvider) {
     ) => {
         try {
             return await provider.execute(
-                snAccount,
+                snAccount as any, // Type cast to avoid version mismatch
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)